.github/workflows/ios-unit-tests.yml

```diff
@@ -57,22 +61,21 @@ jobs:
 
       - name:                 Rust Cache
         uses:                 Swatinem/rust-cache@v2.0.0
-        with:
-          working-directory:  rust
```

Comment:
This is going to break the cache on rust builds and as a result within the same PR _all_ rust deps are going to be rebuilt every time a new commit is pushed to a branch.

Reply:
I've removed it in my file as I was inspecting GA warnings: 
![CleanShot 2022-08-10 at 10 06 20@2x](https://user-images.githubusercontent.com/1955364/183849241-83254bc7-905a-4ddf-a6a8-a586e46257f8.png)

![CleanShot 2022-08-10 at 10 07 20@2x](https://user-images.githubusercontent.com/1955364/183849354-5ea1a71f-9ce2-4587-9ddf-47a72ca09005.png)

Maybe different value should be there @montekki ?